### DISCUSSION TITLE
in latest release: <C-w> weirdly close my current terminal

### DISCUSSION THREAD

**Discussion by cStralpt** [LazyVim/LazyVim#884] [Q&A]
when press C-w in terminal mode it close my window instead of deleting previous word just like in insert mode or "db" in normal mode, on previous version it working fine.

how to reproduce:
- leader-ft
- type something
- press C-w and your cursor will look different
- press C-w again it close your terminal

**Comment by folke** [comment]
I removed the keymap. Forgot to let you know :)
